### PR TITLE
FAQ page for the documentation

### PR DESCRIPTION
**Pull request by mariaboerner1987**
- add a faq page to the documentation
- add first questions to the faq
- add list of Flower meets Blockchain examples

### CODE REVIEW

File: doc/source/faq-section.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
+FAQ
+===
+
+Find here the answers of questions frequently asked.
```
Comment (danieljanes) [inline doc/source/faq-section.rst:4]:
```suggestion
This page collects answers to commonly asked questions about Federated Learning with Flower.
```

File: doc/source/faq-section.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
+FAQ
+===
+
+Find here the answers of questions frequently asked. 
+
+.. dropdown:: :fa:`eye,mr-1` Can I combine federated learning with blockchain?
+
+    Yes, of course. A list of available examples using Flower within a blockchain environment are available here:
```
Comment (danieljanes) [inline doc/source/faq-section.rst:8]:
```suggestion
    Yes, of course. A list of available examples using Flower within a blockchain environment is available here:
```

File: doc/source/faq-section.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+    * `Flower meets Nevermined GitHub Repository <https://github.com/nevermined-io/fl-demo/tree/master/image-classification-flower>`_.
+    * `Flower meets Nevermined YouTube video <https://www.youtube.com/watch?v=A0A9hSlPhKI>`_.
+    * `Flower meets KOSMoS <https://www.kosmos-bmbf.de/wp-content/uploads/sites/13/2021/05/Talk-Flower-Summit-2021.pdf>`_.
+    * `Flower meets Talan blog post <https://www.linkedin.com/pulse/federated-learning-same-mask-different-faces-imen-ayari/?trackingId=971oIlxLQ9%2BA9RB0IQ73XQ%3D%3D>`_ .
+    * `Flower meets Talan GitHub Repository <https://gitlab.com/Talan_Innovation_Factory/food-waste-prevention>`_ .
+
+.. dropdown:: :fa:`eye,mr-1` Does Flower support federated learning on Android devices?
+
+    Yes, it does. Please find have a look to our `blog post <https://flower.dev/blog/2021-12-15-federated-learning-on-android-devices-with-flower>`_ or check out the `example on Flower GitHub <https://github.com/adap/flower/tree/main/examples/android>`_.
```
Comment (danieljanes) [inline doc/source/faq-section.rst:18]:
```suggestion
    Yes, it does. Please take a look at our `blog post <https://flower.dev/blog/2021-12-15-federated-learning-on-android-devices-with-flower>`_ or check out the `Android code example on GitHub <https://github.com/adap/flower/tree/main/examples/android>`_.
```

File: doc/source/faq-section.rst [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+.. dropdown:: :fa:`eye,mr-1` How can I setup Flower on a Raspberry Pi?
```
Comment (danieljanes) [inline doc/source/faq-section.rst:20]:
```suggestion
.. dropdown:: :fa:`eye,mr-1` How can I run Federated Learning on a Raspberry Pi?
```

**Review by danieljanes — CHANGES_REQUESTED** [review]

**Comment by mariaboerner1987** [comment]
@danieljanes changes are included.

File: doc/source/index.rst [hunk 5]
```diff
@@ -31,6 +31,7 @@ Flower's design goals was to make this simple. Read on to learn more.
    fedbn-example-pytorch-from-centralized-to-federated
    recommended-env-setup
    changelog
+   faq-section
```
Comment (danieljanes) [inline doc/source/index.rst:34]: @mariaboerner1987 last change: can we rename this to just `faq` to make the URL nicer?

**Review by danieljanes — CHANGES_REQUESTED** [review]

File: doc/source/index.rst (hunk 5 above)
Comment (mariaboerner1987) [reply doc/source/index.rst]: - renamed the file
- change the name in index.rst

**Review by danieljanes — APPROVED** [review]

**Pull request merged**